pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```
@@ -38,40 +50,35 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
+  ];
 
-  strictDeps = true;
-
-  buildPhase = ''
-    runHook preBuild
+  ignoreCompilationError = false;
 
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
+  strictDeps = true;
 
-    runHook postBuild
+  postPatch = ''
+    lispdir=$out/share/emacs/site-lisp/elpa/cask-${finalAttrs.version} \
```

Not necessary for a stable version, but `melpaVersion` works for unstable versions, too.
```suggestion
    lispdir=$out/share/emacs/site-lisp/elpa/cask-${finalAttrs.melpaVersion} \
```

---

Done.